apps/hannk/tflite/CMakeLists.txt

```diff
@@ -13,6 +13,11 @@ set(TFLITE_ENABLE_RUY OFF)
 set(TFLITE_ENABLE_XNNPACK OFF)
 
 # Also have to disable some stuff dragged in by TFLite
+if(POLICY CMP0077)
+  # This won't do the trick -- we need to use CMAKE_POLICY_DEFAULT_CMP0077, alas
+  # cmake_policy(SET CMP0077 NEW)
+  set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+endif()
```

Comment:
No need to test (`if`) for the policy before setting it because the minimum is 3.16, which means the policy exists. This line should also be moved to the top-level, beneath the `project()` call in `apps/hannk/CMakeLists.txt`.

Reply:
done